go.mod

```diff
@@ -1,9 +1,15 @@
 module github.com/bitrise-steplib/steps-cache-pull
 
-go 1.16
+go 1.17
```

Comment:
This should be the minimum go version the step runs correctly on. Is this the minimum working version?

Reference: https://go.dev/doc/modules/gomod-ref

Reply:
Technically no, but our convention is to keep this in sync with the Go version installed on all supported stacks (which is 1.17 at the moment). I bumped this version number because starting with 1.17, `go mod tidy` adds the transitive dependencies to this file too, which is a good idea I think.